docs/docs/users/filecoin_services.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Filecoin Services
+hide_title: true
+sidebar_position: 6
+---
+
+<p align="center">
+  <img src="/img/logo.png" alt="Forest logo"/>
+  <img src="/img/chainsafe_logo.png" alt="ChainSafe logo"/>
+  <img src="/img/filecoin_logo.png" alt="Filecoin logo"/>
+</p>
+
+ChainSafe works on various projects in the Filecoin ecosystem. While the Forest team focuses on improving the Forest client and collaborating with other Filecoin implementations to drive the network forward, ChainSafe's Infrastructure team provides a number of services to the Filecoin community.
+
+## 🌲Forest
```

Comment:
I'm not sure it makes sense for Forest to be at the top of this, nor to separate services into forest vs infra. The purpose of this page is to promote the services we operate, shilling Forest should be secondary. Also the distinction of what each team operates is probably not relevant to users (unless they have issues...)

I would suggest mentioning Forest in a callout (ie. admonition, `:::info....`), collapsing this into a single list, and moving the status page to a callout as well.

Reply:
> Also the distinction of what each team operates is probably not relevant to users (unless they have issues...)

That's the exact reason for this distinction, given folks were quite often reaching out to Forest team, and the only thing we could do is to forward it to the infra team. It's not a huge hassle, though.

I can de-prioritize Forest in this page.